### PR TITLE
Create pixhack_V3.md

### PR DESCRIPTION
**Pull request by cuavbin**

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thank you very much for this. Can you please move the file from "在/flight_controller/pixhack_V3.md" to en/flight_controller/pixhack_v3.md" (note, the use of **en** as the root, meaning "English".

File: 在/flight_controller/pixhack_V3.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Pixhack V3
+
+The [Pixhack V3] autopilot is a flexible autopilot intended primarily for manufacturers of commercial systems.  It is based on the [Pixhawk-project](https://pixhawk.org/) **FMUv3** open hardware design and runs PX4 on the [NuttX](http://nuttx.org) OS.
+
+<img src="http://doc.cuav.net/PixHack/assets/pixhackv31.png" width="600px" />
```
Comment (hamishwillee) [inline 在/flight_controller/pixhack_V3.md]: Is it OK if I download and include the file locally in the build?

**Comment by hamishwillee** [comment]
Other information it would be useful if you could provide:
- "Availability"  - is there a good place to buy this?
- Links to setup instructions - ie how you wire it up
- Detail about the hardware or links to that - if you checkout [this example](https://docs.px4.io/en/flight_controller/mro_pixhawk.html) you can see it covers details about the microprocessor, sensors, interfaces, powering, and weight/dimensions. The idea is to help people understand at a glance what the microprocessor offers. 

Do you think you might be able to add some information like that?

**Comment by cuavbin** [comment]
Thanks for your advice, I am fixing it
Please help me move to EN folder

**Comment by hamishwillee** [comment]
Thanks @cuavbin ! I have replaced this PR with #119. Can you please see my comments there.

This is now closed

**Pull request closed without merging**